katran/lib/Netlink.cpp

```diff
@@ -173,6 +174,35 @@ NetlinkMessage NetlinkMessage::TC(
   return ret;
 }
 
+NetlinkMessage NetlinkMessage::QD(unsigned ifindex) {
+  NetlinkMessage ret;
+  unsigned char* buf = ret.buf_.data();
+
+  struct nlmsghdr* nlh;
+  struct tcmsg* tc;
+  uint32_t protocol = 0;
+  unsigned int bpfFlags = TCA_BPF_FLAG_ACT_DIRECT;
```

Comment:
Do you plan to use these two variables in a future diff?

Reply:
nope. remenants of copy-paste. will remove